app/components/ui/Navigation.tsx

```diff
@@ -28,19 +79,4 @@ export const Navigation = ({
   );
 };
 
-const SiteLogo = () => (
-  <Link className={`${styles.navLogo}`} to="/">
-    {/* TODO */}
-    [SITE LOGO HERE]
-  </Link>
-);
-
-const SiteLinks = () => {
-  return (
-    <div className={styles.navRight}>
-      <Translate />
-    </div>
-  );
-};
-
```

Comment:
It could be worth separating these out later but I didn't see much use for this abstraction at the moment while I was rewriting things.